articles/dms/tutorial-mysql-azure-mysql-offline-portal.md

```diff
@@ -0,0 +1,254 @@
+---
+title: "Tutorial: Migrate MySQL offline to Azure Database for MySQL"
+titleSuffix: "Azure Database Migration Service"
+description: "Learn to perform an offline migration from MySQL on-premises to Azure Database for MySQL by using Azure Database Migration Service."
+services: dms
+author: "sumitgaurin"
+ms.author: "sgaur"
+manager: "arthiaga"
+ms.reviewer: "arthiaga"
+ms.service: dms
+ms.workload: data-services
+ms.custom: "seo-lt-2019"
+ms.topic: tutorial
+ms.date: 04/11/2021
+---
+
+# Tutorial: Migrate MySQL to Azure Database for MySQL offline using DMS
```

Comment:
Please review my comments on pre-reqs and so on for the PowerShell article and make similar updates in this article.

Reply:
Done